### ISSUE TITLE
accuracy score

### ISSUE THREAD

**Issue by nelzaatari** [iubh/DLMDSML01#2]
good day prof,

in the notebook
https://github.com/iubh/DLMDSML01/blob/main/Q_A_Sessions/01_intro_to_ml/01_intro_to_ml.ipynb

the accuracy score 

acc_nn1.append(metrics.accuracy_score(yhat_nn1 , y_test))

must be 

acc_nn1.append(metrics.accuracy_score(y_test , yhat_nn1))

becuause according to the documentation in sklean https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html

first parameter is y_true,second parameter y_pred

or maybe there is something that i'm confused about

moreover, should we average the the accuracy results of each classifier ?

thank you prof,
